src/data_morph/shapes/bases/line_collection.py

```diff
@@ -76,7 +77,7 @@ def _distance_point_to_line(
         .. _this VBA code: http://local.wasp.uwa.edu.au/~pbourke/geometry/pointline/source.vba
         """
         start, end = line
-        line_mag = self._euclidean_distance(start, end)
+        line_mag = self._euclidean_distance(np.array(start), np.array(end))
```

Comment:
`np.array(start)`, `np.array(end)`, and `np.array(point)` could be stored in variables to only do that once here.